devel/git-sizer/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            github git-sizer 1.0.0 v
+categories              devel
+platforms               darwin
+supported_archs         noarch
+maintainers             @gerardsoleca openmaintainer
+license                 MIT
+
+description             Compute various size metrics for a Git repository, \
+                        flagging those that might cause problems
+long_description        ${description}
+
+github.tarball_from     releases
+distname                ${name}-${version}-${platforms}-amd64
```

Comment:
It would seem like you're unpacking the pre-built version here rather than depending on the `go` package and building from source for the platform? In general, we prefer building from source unless there's really no choice.